src/bootstrap/src/utils/helpers.rs

```diff
@@ -421,29 +421,18 @@ pub fn linker_flags(
     builder: &Builder<'_>,
     target: TargetSelection,
     lld_threads: LldThreads,
-    stage: u32,
+    _stage: u32,
 ) -> Vec<String> {
     let mut args = vec![];
     if !builder.is_lld_direct_linker(target) && builder.config.lld_mode.is_used() {
         match builder.config.lld_mode {
             LldMode::External => {
-                // cfg(bootstrap) - remove the stage 0 check after updating the bootstrap compiler:
-                // `-Clinker-features` has been stabilized.
-                if stage == 0 {
-                    args.push("-Zlinker-features=+lld".to_string());
-                } else {
-                    args.push("-Clinker-features=+lld".to_string());
-                }
+                args.push("-Zlinker-features=+lld".to_string());
+                args.push("-Clinker-features=+lld".to_string());
```

Comment:
Problem: first one is wrong, it should be stable now

Reply:
Also it would be nice to remove the `stage` parameter, it will need to be changed on a bunch of places though to remove the propagated parameter :innocent:

Reply:
is it not worth just keeping it around ~indefinitely? It seems annoying for folks to have to constantly re-add the same "boilerplate" to pass stage information down here and then remove it again every time we start using an unstable flag

Reply:
I don't think we needed it here before, the previous occurrence was IIRC my fail because I did something wrong there. Even if it becomes required again in the future, I don't personally like keeping unused stuff around.